src/component-library/Accordion/Accordion.style.tsx

```diff
@@ -28,7 +28,7 @@ const StyledAccordionItemWrapper = styled.div<Pick<StyledAccordionProps, '$isDis
 
 const StyledAccordionItemHeading = styled(H3)`
   margin: 0;
-  font-weight: ${theme.fontWeight.bold};
```

Comment:
Discussed these changes with Jay—he wants the changes to be consistent across all accordions, rather than introduce a variant.